dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/DataManagerTest.kt

```diff
@@ -106,6 +99,7 @@ class DataManagerTest(
             storableEuTaxonomyDataSetForNonFinancials, correlationId,
         )
         `when`(mockStorageClient.selectDataById(dataId, correlationId)).thenReturn("")
+        dataManager.removeStoredItemFromTemporaryStore(dataId, "", MessageType.DataStored)
```

Comment:
Isn't that redundant? BeforeEach the dataManager will be reset, and then it's again guaranteed that the dataId is removed.

Reply:
This is to prevent running into the case that the data is still in the map and the StorageControllerApi is actually used.

Reply:
ok